tests/_exe/do_while.c

```diff
@@ -12,6 +12,18 @@ int bad_factorial(int num) {
 }
 
 void main() {
-  printf("Factorial of 5: %d\n", bad_factorial(5));
-  printf("Factorial of 10: %d\n", bad_factorial(10));
+  int fac5 = factorial(5);
+  int fac10 = factorial(10);
+
+  if (fac5 == 120) {
+    putchar('O');
+  } else {
+    putchar('B');
+  }
+
+  if (fac10 == 3628800) {
+    putchar('O');
+  } else {
+    putchar('B');
+  }
```

Comment:
Since `tests/_exe/do_while.c` and `tests/_sh/do_while.c` are the same modulo `printf`, I think we can share the same test file for both backend and move it to `tests/_all/do_while.c`

Reply:
Oh! Was not aware!